CONTRIBUTING.md

```diff
@@ -180,23 +180,20 @@ $ git push origin [name_of_your_new_branch]
 
 
 ### Setting up for cmd2 development
-For doing cmd2 development, you actually do NOT want to have cmd2 installed as a Python package.
-So if you have previously installed cmd2, make sure to uninstall it:
+For doing cmd2 development, it is recommended you create a virutal environment and install the package from the source.
 ```sh
-$ pip uninstall cmd2
-```
+# Create a new environment for cmd2
+$ conda create -n cmd2 python=3.6
```

Comment:
Recommend including Python version in name of conda env in case developers wish to test cmd2 with more than one version of Python
- e.g. cmd2_py36